datalad/cli/common_args.py

```diff
@@ -34,10 +34,13 @@
         ('-c',),
         dict(action='append',
              dest='cfg_overrides',
-             metavar='KEY=VALUE',
-             help="""configuration variable setting. Overrides any
-             configuration read from a file, but is potentially overridden
-             itself by configuration variables in the process environment.
+             metavar='[:]name[=value]',
```

Comment:
would it work if `:name=value` and what would be the effect?  I just wonder if should be made more explicit like e.g.

```suggestion
             metavar='(:name|name=value)',
```
?